pythonlib/src/mill/pythonlib/PythonModule.scala

```diff
@@ -75,6 +75,9 @@ trait PythonModule extends PipModule with TaskModule { outer =>
    */
   def unmanagedPythonPath: T[Seq[PathRef]] = Task { Seq.empty[PathRef] }
 
+  def generatedSources: T[Seq[PathRef]] = Task { Seq.empty[PathRef] }
```

Comment:
You can include a comment what this does. I think it's fine to reuse the one from JavaModule
```
/**
   * Folders containing source files that are generated rather than
   * handwritten; these files can be generated in this target itself,
   * or can refer to files generated from other targets
   */
```

Reply:
Actually i haven't updated the docs yet, i do that after code satisfaction 🙂